pkg/test/echo/common/response/response.go

```diff
@@ -20,7 +20,8 @@ import (
 )
 
 var (
-	StatusCodeOK = strconv.Itoa(http.StatusOK)
+	StatusCodeOK        = strconv.Itoa(http.StatusOK)
+	StatusCodeForbidden = strconv.Itoa(http.StatusForbidden)
```

Comment:
The reason I remove this from here is because it's RBAC specific ... it's not used by this class.